### PR TITLE
remove unused wiremock instance

### PR DESCRIPTION
**Pull request by songkaiape**

### CODE REVIEW

**Comment by vscjavaci** [comment]
Can one of the admins verify this patch?

File: Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java [hunk 1]
```diff
@@ -92,23 +88,6 @@ public void setUpStep() throws Exception {
 
         addTextReplacementRule(GLOBAL_ENDPOINT, MOCK_URI + "/");
 
-        int retries = 10;
-        boolean created = false;
-        while (retries > 0) {
-            retries--;
-            try {
-                wireMock = new WireMock(MOCK_HOST, wireMockRule.port());
```
Comment (adashen) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: no host specified?

Comment (adashen) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: I remember in reset, wireMock is set to null. So when will it be created after one test case is run?

Comment (songkaiape) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: wiremock will set localhost as default, if we need to run case on other host, then I should add Mock_Host.

Comment (songkaiape) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: Yes, @rule will generate a new one for each test case. Also I think I should delete the wireMock=null .No need for that.

File: Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java [hunk 2]
```diff
@@ -153,7 +132,6 @@ protected void resetTest(String testName) throws Exception {
         if (!currentTestName.equals(testName)) {
             return;
         }
-        wireMock=null;
```
Comment (adashen) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: no need to reset mapping?

Comment (songkaiape) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: I have test on my local machine , everything works good. The mapping should be reset when a new wiremock instance is generated.

Comment (adashen) [inline Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java]: Is there api in wiremockrule to reset mapping? if yes i think we need to add it.

**Comment by songkaiape** [comment]
retest this please.

**Comment by adashen** [comment]
retest this please.

**Comment by adashen** [comment]
retest this please.

**Pull request merged**